flutter: improve session and screenshots docs (#11708)
getsentry/sentry-docs: platform-includes/configuration/auto-session-tracking/flutter.mdx

To benefit from the health data, you must use at least version 4.0.0 of the Flutter SDK.

#### Session Timeout

> By default, the session is terminated once the application is in the background for more than `30 seconds`. You can change the default timeout with the option `autoSessionTrackingInterval` to a `Duration` of your choosing.

Reviewer: ```suggestion
By default, the session is terminated once the application is in the background for more than `30 seconds`. You can change this default by setting the `autoSessionTrackingInterval` option to a `Duration` of your choosing.
```

```dart
SentryFlutter.init((options) {
  // Change timeout duration until session is terminated in background
  options.autoSessionTrackingInterval = const Duration(seconds: 60)

> }, appRunner: () => runApp(MyApp()));

Author: I am unsure whether we should leave the appRunner in there, it is an optional parameter

```

#### Disable Auto Session Tracking

> If you'd like to opt out of capturing sessions, set the option `enableAutoSessionTracking` to `false`. If you disable this feature, release health will not be available.

Reviewer: ```suggestion
If you'd like to opt out of capturing sessions, set the `enableAutoSessionTracking` option to `false`. If you disable this feature, release health will not be available.
```
